Packs/NetskopeV2/Integrations/NetskopeV2/NetskopeV2.py

```diff
@@ -20,33 +22,34 @@ class Client(BaseClient):
 
     def get_lists(self):
         return self._http_request(
-            method='GET'
+            method='GET',
+            url_suffix='/api/v2/policy/urllist'
         )
 
     def get_list(self, list_id):
         return self._http_request(
             method='GET',
-            url_suffix=list_id
+            url_suffix=f'/api/v2/policy/urllist/{list_id}'
         )
 
     def patch_list(self, list_id, url_list_object, action):
         return self._http_request(
             method='PATCH',
-            url_suffix=f'{list_id}/{action}',
+            url_suffix=f'/api/v2/policy/urllist/{list_id}/{action}',
             json_data=url_list_object
         )
 
     def replace_url_list(self, list_id, url_list_object):
         return self._http_request(
             method='PUT',
-            url_suffix=list_id,
+            url_suffix=f'/api/v2/policy/urllist/{list_id}',
             json_data=url_list_object
         )
 
     def deploy_lists(self):
         return self._http_request(
             method='POST',
-            url_suffix='deploy'
+            url_suffix='/api/v2/policy/urllist/deploy'
```

Comment:
if the suffix exist in each api call, why did you remove it from the base url and added it to each call?

Reply:
The purpose for these changes is because Netskope modified their API about two weeks ago. From Netskope: "As the number of endpoints on REST API v2 platform grows, we had to tighten up the route matching rules in REST API v2 API gateway. Therefore some ways of using REST API v2 in not quite documented manner, including use of trailing slash, are no longer accepted by API gateway. For trailing slash specifically, we have to enforce this rule as the acceptance of trailing slash varies by various server side frameworks, and there is a variety of different server framework used by different backend services at Netskope. So we had to disallow it to avoid customer confusion around inconsistency of uses for various different endpoints on REST API v2”

Reply:
the get_lists() function would fail every time because the URL ended in /api/v2/policy/urllist/ (note the ending forward slash which Netskope modified their API gateway to reject)